src/jit/lclvars.cpp

```diff
@@ -6491,6 +6601,14 @@ void Compiler::lvaDumpEntry(unsigned lclNum, FrameLayoutState curState, size_t r
     {
         printf(" stack-byref");
     }
+    if (varDsc->lvClassHnd != nullptr)
+    {
+        printf(" class-hnd");
```

Comment:
Did you want to actually dump the class handle (or look up a name for it and dump that)?